mobile/lib/repositories/integration.repository.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:http/http.dart' as http;
+import 'package:triggo/api/call.dart';
+import 'package:triggo/api/response.dart';
+import 'package:triggo/repositories/credentials.repository.dart';
+import 'package:triggo/repositories/dtos/integration.dtos.dart';
+
+class IntegrationRepository {
+  final http.Client? client;
+  final CredentialsRepository? credentialsRepository;
```

Comment:
Let's say that you MUST give a credential repository
```suggestion
  final CredentialsRepository credentialsRepository;
```